packages/relay/src/lib/eth.ts

```diff
@@ -222,6 +252,25 @@ export class EthImpl implements Eth {
     return EthImpl.numberTo0x(fee);
   }
 
+  private getRepeatedFeeHistory(blockCount: number, oldestBlockNumber: number, rewardPercentiles: Array<number> | null, fee: string) {
+    const shouldIncludeRewards = Array.isArray(rewardPercentiles) && rewardPercentiles.length > 0;
+
+    const feeHistory = {
+      baseFeePerGas: Array(blockCount).fill(fee),
+      gasUsedRatio: Array(blockCount).fill(EthImpl.defaultGasUsedRatio),
+      oldestBlock: EthImpl.numberTo0x(oldestBlockNumber),
+    };
+
+    // next fee
```

Comment:
```suggestion
    // next fee. Due to high block production rate and low fee change rate we add the next fee since by the time a user utilizes the response there will be a next block likely with the same fee
```